apps/docs/content/components/code/sizes.ts

```diff
@@ -2,11 +2,11 @@ const App = `import {Code} from "@nextui-org/react";
 
 export default function App() {
   return (
-    <div className="flex gap-4">
+    <div className="flex flex-col gap-4">
```

Comment:
This is what we use for `Snippet`
```suggestion
    <div className="flex flex-wrap gap-4">
```